lib/users.js

```diff
@@ -8,12 +11,45 @@ const schema = yup.object().shape({
   password: yup.string().required().min(8)
 });
 
-users.signUp = (userInfo) => {
+users.signUp = async (userInfo) => {
   return schema.validate(userInfo).catch((err) => { return {
       errors: err.errors
     }
   })
 } 
 
+users.logIn = async (userInfo) => {
+  try {
+    await schema.validate(userInfo)
```

Comment:
remove. User input may be a username or email.

Reply:
you can just do a simple check:

```
if (!userInfo.username && !userInfo.email) {
  logger.info('invalid userInfo for login', userInfo)
  return {
    errors: ['invalid input']
  }
```

Reply:
That's a good idea